### DISCUSSION TITLE
How can I  use this extension  for foreign tables  since  primary key constraints are not supported on foreign tables  .

### DISCUSSION THREAD

**Discussion by ginjojohny** [supabase/supabase#6564] [Questions]
I am working with a schema that have foreign tables which are created by using Oracle  FDW . How can I use supa_audit  extension with these tables  since  primary  key constraints are not supported on foreign tables  . Can you suggest any workarounds for it  or is it  impossible ?

**Comment by steve-chavez** [comment]
@ginjojohny I suggest you report your issue on https://github.com/supabase/supa_audit.
